JustSaying/Messaging/MessageSerialisation/MessageSerialisationRegister.cs

```diff
@@ -1,57 +1,71 @@
 using System;
+using System.Collections.Concurrent;
 using System.Collections.Generic;
 using JustSaying.Models;
 
 namespace JustSaying.Messaging.MessageSerialisation
 {
     public class MessageSerialisationRegister : IMessageSerialisationRegister
     {
+        // TODO Inconsistent use of American and British English for "serialize"/"serialise"
```

Comment:
I think we should standardise (or is it standardize 😉) on the american spelling, we're fighting with the ecosystem otherwise

Reply:
Personally I go with the _zees_ to match tooling defaults, but I'm happy to go with the _esses_ if that's the majority opinion. We just need to be consistent.

Reply:
Same feelings here, if the APIs are getting a rehaul then count me as a vote for the *zees*, if not then these methods should match.

Reply:
I vote for _zees_ for the reasons @slang25 mentioned.

Reply:
#457